### ISSUE TITLE
Luchi programa IA

### ISSUE THREAD

**Issue by Luckyblocky2** [Luckyblocky2/LUCKYBLOCKYS#1]
import tensorflow as tf
from tensorflow.keras.datasets import mnist
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Flatten
import matplotlib.pyplot as plt

# Cargar el conjunto de datos MNIST (dígitos escritos a mano)
(x_train, y_train), (x_test, y_test) = mnist.load_data()

# Normalizar los valores de píxeles entre 0 y 1
x_train, x_test = x_train / 255.0, x_test / 255.0

# Crear el modelo de RNA
model = Sequential([
    Flatten(input_shape=(28, 28)),  # Aplanar la imagen
    Dense(128, activation='relu'),   # Capa oculta con 128 neuronas y función de activación ReLU
    Dense(10, activation='softmax')  # Capa de salida con 10 neuronas (una por cada dígito) y función de activación softmax
])

# Compilar el modelo
model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

# Entrenar el modelo
model.fit(x_train, y_train, epochs=5)

# Evaluar el modelo en el conjunto de prueba
test_loss, test_acc = model.evaluate(x_test, y_test)
print(f"Precisión en el conjunto de prueba: {test_acc:.4f}")

# Hacer predicciones
predictions = model.predict(x_test)

# Visualizar algunas predicciones
plt.figure(figsize=(10, 10))
for i in range(25):
    plt.subplot(5, 5, i + 1)
    plt.imshow(x_test[i], cmap='gray')
    plt.title(f"Predicción: {tf.argmax(predictions[i])}")
    plt.axis('off')
plt.show()                                  como puedo generar esta IA

**Comment by Luckyblocky2** [comment]
Cuanto es 2+2
